### PR TITLE
Mega Menu Updates

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jashpatel1** [comment]
@leecalcote I wish to include a few more things in this PR. Maybe we can merge this after today's meeting.

**Comment by leecalcote** [comment]
Sounds real good. Speaking of today's meeting, I'm going to be detained by ServiceMeshCon / KubeCon, and so, I'll ask @chandrashritii to organize and run today's meeting.

File: src/sections/Navigation/utility/data.js [hunk 1]
```diff
@@ -39,19 +39,19 @@ const Data = {
             subItems: [
                 {
                     name: "Meshery",
-                    path: "#"
+                    path: "/projects/meshery"
```
Comment (leecalcote) [inline src/sections/Navigation/utility/data.js:42]: How do we ensure that the `/meshery` path works as well?

Comment (Jashpatel1) [reply src/sections/Navigation/utility/data.js:42]: We can have redirect paths in the mdx file for meshery and others, similar to the one we do in jekyll.

Comment (leecalcote) [reply src/sections/Navigation/utility/data.js:42]: Sounds real good

**Pull request merged**